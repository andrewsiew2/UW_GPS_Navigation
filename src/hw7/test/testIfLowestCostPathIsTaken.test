# Find a path with Dijkstra's algorithm when there are 2 possible paths
# tests if it takes the lowest cost path

LoadGraph test testing.tsv

FindPath test SPECTRE BARA
