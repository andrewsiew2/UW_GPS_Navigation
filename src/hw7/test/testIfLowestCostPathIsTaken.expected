# Find a path with Dijkstra's algorithm when there are 2 possible paths
# tests if it takes the lowest cost path

loaded graph test

path from SPECTRE to BARA:
SPECTRE to MORTRED with weight 0.500
MORTRED to BARA with weight 1.000
total cost: 1.500